CHANGELOG.md

```diff
@@ -1,6 +1,8 @@
 # Changelog
 
 ## [unreleased]
+* Making Veroivo thread safe (@eNote-GmbH)
```

Comment:
Fix typo. Next time make a distinct PR with [skip-ci] for changelog updates. Thanks.